automates/program_analysis/CAST2GrFN/visitors/cast_visitor.py

```diff
@@ -0,0 +1,42 @@
+from automates.utils.method_dispatch import methdispatch
+from automates.program_analysis.CAST2GrFN.model.cast import (
+    AstNode,
+    Assignment,
+    Attribute,
+    BinaryOp,
+    BinaryOperator,
+    Call,
+    ClassDef,
+    Dict,
+    Expr,
+    FunctionDef,
+    List,
+    Loop,
+    ModelBreak,
+    ModelContinue,
+    ModelIf,
+    ModelReturn,
+    Module,
+    Name,
+    Number,
+    Set,
+    String,
+    Subscript,
+    Tuple,
+    UnaryOp,
+    UnaryOperator,
+    VarType,
+    Var,
+)
+
+
+class CASTVisitor:
+    def __init__(self):
+        pass
+
+    def visit_list(self, node_list: List):
+        return [self.visit(n) for n in node_list]
+
+    @methdispatch
+    def visit(self, node: AstNode):
```

Comment:
@dpdicken see my previous comment in `cast_to_air_visitor.py` ... this error message works for now, but once all expected `AstNode`s are handled please convert this to a `TypeError`.